src/pages/LoginPage.js

```diff
@@ -22,7 +25,7 @@ export default function LoginPage({setAuth}) {
       password
     }
 
-    const response = await fetch('http://localhost:5000/auth/login',{
+    const response = await fetch(`${process.env.REACT_APP_BASE_URL}/auth/login`,{
```

Comment:
Niiiice, now does not matter if its localhost or aws or azure or... you get the point :)